app/components/login.jsx

```diff
@@ -1,202 +1,318 @@
+                        <input
+                            className={classes.checkTermBox}
+                            type='checkbox'
+                            name='agreed'
+                            onClick={e => methods.onChangeAgree(e.target.checked)}
+                        />
+                        <label className={classes.agreeTermLabel}>
+                            I agree to the
+                            <a href='https://enciv.org/terms' target='_blank' className={classes.aLinkTerm}>
+                                Terms of Service
+                            </a>
+                        </label>
+                    </div>
+                </div>
+                <div className={classes.btnContainer}>
+                    <button className={cx(classes.btn, isLogIn && classes.disabled)} onClick={e => methods.signup()}>
+                        Sign Up
+                    </button>
+                    <button className={cx(classes.btn, !isLogIn && classes.disabled)} onClick={e => methods.login()}>
+                        Log In
+                    </button>
+                    <button className={cx(classes.btn, !isLogIn && classes.disabled)} onClick={e => methods.skip()}>
+                        Skip
+                    </button>
+                </div>
+                <div className={cx(classes.resetPasswordBtn, !isLogIn && classes.disabled)}>
+                    <button onClick={e => methods.sendResetPassword()} className={classes.resetBtn}>
+                        Send Reset Password
+                    </button>
+                </div>
+                <div className={cx("go-to-sign-up", isLogIn && classes.disabled)}>
+                    <p> Don't have an account? Click Join, it only takes a minute to sign up!</p>
+                </div>
+                <div>
+                    {state.error && <div style={{ color: '#fec215', textAlign: 'center' }}>{state.error}</div>}
+                    {state.info && <div style={{ color: '#fec215', textAlign: 'center' }}>{state.info}</div>}
+                    {state.success && <div style={{ color: '#fec215', textAlign: 'center' }}>{state.success}</div>}
+                </div>
+            </div>
+        </div>)
 }
 
-export default Login;
+export default React.forwardRef(Login);
+
+const useStyles = createUseStyles(theme => ({
+    Login: {
+        backgroundColor: 'white', 
+        width: '60rem',
+        maxWidth: '100vw',
+        margin: 0,
+        borderRadius: '1rem',
+        height: '100%',
+        padding: '0',
+        fontFamily: 'Inter',
+        fontSize: '16px', // change to rem?
+        fontWeight: '700',
+        lineHeight: '16px', // change to rem?
+        textAlign: 'center',
+        position: 'fixed',
+        left: '50%',
+        top: '50%',
+        transform: 'translate(-50%,-50%)',
+        zIndex: 10,
+        maxHeight: '90vh',
+        overflowY: 'auto',
+    },
+    btnClick: {
+        color: '#1A1A1A',
```

Comment:
Please use colors out of theme. If this color isn't there - put it there.  Same for all other colors.

Reply:
I'm a bit unsure about this. The only color I know that I can use is theme.colorPrimary which is black, but are you saying that any other color that is not black can be left as is? And that anything with a dark shade should be replaced with theme.colorPrimary

Reply:
go look in app/components/theme.js there are many colors.  If there isn't one for #1A1A1A then add it.